mlflow/tracking/client.py

```diff
@@ -107,6 +107,30 @@ def get_run(self, run_id):
 
         :return: A single :py:class:`mlflow.entities.Run` object, if the run exists. Otherwise,
                  raises an exception.
+
+        .. code-block:: python
+            :caption: Example
+
+            import mlflow
+            from mlflow.tracking import MlflowClient
+
+            with mlflow.start_run() as run:
+                mlflow.log_param("p", 0)
```

Comment:
```suggestion
            with mlflow.start_run():
                mlflow.log_param("p", 0)
```

Reply:
"run" not used, hence not needed.

Reply:
Sorry I missed this line `run = client.get_run(run.info.run_id)`.